app/models/katello/module_stream.rb

```diff
@@ -56,6 +56,22 @@ def self.available_for_hosts(hosts)
             merge(::Katello::Host::ContentFacet.where(host_id: hosts)))
     end
 
+    def self.with_name_streams(name_streams)
+      return where("1=0") if name_streams.blank?
+      ns = name_streams.shift
+      query = where(name: ns.first, stream: ns.last)
+      name_streams.each do |name, stream|
+        query = query.or(where(name: name, stream: stream))
+      end
+      query
+    end
+
+    def self.uniquify_by_name_streams(name_streams)
```

Comment:
where is this used?

Reply:
Good call. Going to remove it (left over stray method from the name/stream implementation).